crates/sovereign-sdk/adapters/mock-da/src/service.rs

```diff
@@ -380,10 +380,10 @@ impl DaService for MockDaService {
         ([0u8; 32], ())
     }
 
-    async fn send_transaction(&self, blob: &[u8]) -> Result<(), Self::Error> {
+    async fn send_transaction(&self, blob: &[u8]) -> Result<Self::TransactionId, Self::Error> {
         let blocks = self.blocks.lock().await;
         let _ = self.add_blob(&blocks, blob, Default::default())?;
-        Ok(())
+        Ok(MockHash::from([0; 32]))
```

Comment:
if we're gonna have a transaction id in mock da, can you use that in txs_commitment field in mock block?

can just put in tx id for the field instead of blockhash